docs/guides.rst

```diff
@@ -140,7 +141,7 @@ Individual Simulation settings
 -------------------------------
 
 To customise the specifics of how each lambda window is run (e.g. timestep), you can use the ``set_simfile_option`` method. For example, to set the timestep to 2 fs, run
-``calc.set_simfile_option("timestep", "2 * femtosecond")``. This will change parameters from the defaults given in ``template_config.cfg`` in the ``input`` directory, and warn
+``calc.set_simfile_option("timestep", "2 * femtosecond")``. This will change parameters from the defaults generated by ``engine_config``, and warn
```

Comment:
We need to update this entire function to update the configs for all sub-simulation runners. Please rename it to something more appropriate e.g. set_config_option and test that it updates all configs.